packages/api/src/repository/Feed.ts

```diff
@@ -16,16 +16,11 @@ export class FeedRepository {
       {}
     )
 
-    const sortedFeedsWithoutEth = dataFeeds
-      .filter(feed => !feed.network.includes(Network.EthereumMainnet))
-      .sort((a, b) => (b.network < a.network ? 1 : -1))
+    const sortedFeeds = dataFeeds.sort((a, b) =>
+      b.network < a.network ? 1 : -1
+    )
 
-    this.sortedDataFeeds = [
-      ...this.dataFeedsByNetwork[Network.EthereumMainnet],
-      ...this.dataFeedsByNetwork[Network.EthereumGoerli],
-      ...this.dataFeedsByNetwork[Network.EthereumRinkeby],
-      ...sortedFeedsWithoutEth
-    ]
+    this.sortedDataFeeds = [...sortedFeeds]
```

Comment:
I think we should be able to assign directly to `this.sortedDataFeeds` without the spreading. That will be way more efficient :wink: